components/Participant.tsx

```diff
@@ -31,16 +35,20 @@ const Participant = ({ gain, participant, isYou }) => {
       track && playTrack(track);
     });
 
-    participant.on('trackSubscribed', track => {
+    const onTrackSubscribed = track => {
       taggedLog(`trackSubscribed (${track.kind})`, track);
       playTrack(track);
-    });
+    };
+
+    participant.on('trackSubscribed', onTrackSubscribed);
+    return () => {
+      audioCtx.close();
```

Comment:
Closing the `AudioContext` ensures that we don't hit the per-tab `AudioContext` limit (which is I think 6 contexts/tab) when hot-reloading the app.